### PR TITLE
Add profile photo to users model

### PR DESCRIPTION
**Pull request by enzofab91**
### **Feature:**
As a user I should be able to upload a picture to my profile.

### **Trello reference:**
https://trello.com/c/GaAueZvo

### **Description:**
This PR adds a profile photo to users with [active_storage_base64](https://github.com/rootstrap/active-storage-base64) gem and [Active Storage](https://edgeguides.rubyonrails.org/active_storage_overview.html).

### CODE REVIEW

File: spec/requests/api/v1/users/update_spec.rb [hunk 1]
```diff
@@ -1,13 +1,22 @@
+require 'rails_helper'
```
Comment (martinjaimem) [inline spec/requests/api/v1/users/update_spec.rb:1]: are we sure this is necessary?

Comment (enzofab91) [reply spec/requests/api/v1/users/update_spec.rb:1]: Odd, sometimes if I don't write it explicitly it won't work. In this particular case, it raises _undefined method 'create'_ when creating user.

Comment (martinjaimem) [reply spec/requests/api/v1/users/update_spec.rb:1]: you are right 👍  this project doesn't have the `.rspec` file.

I recommend you add it, it's just a file at the root of the project named `.rspec` that should have
```
--require rails_helper
```
as it content.

That way rspec always loads that file before running specs 😉

**Review by martinjaimem — APPROVED** [review]
LGTM

File: spec/requests/api/v1/users/update_spec.rb (hunk 1 above)
Comment (enzofab91) [reply spec/requests/api/v1/users/update_spec.rb:1]: Excellent! 🚀

**Review by sebastiancaraballo — COMMENTED** [review]
As discussed, try refactoring into [current active storage implementation](https://edgeguides.rubyonrails.org/active_storage_overview.html#has-one-attached).

**Review by sebastiancaraballo — APPROVED** [review]

**Pull request merged**